spec/tictactoe_spec.rb

```diff
@@ -7,7 +7,25 @@
   let (:test_ttt) { TicTacToe.new(game_io, player1, player2) }
 
   it "runs the game loop " do
-    allow(test_ttt.game_loop).to receive(:game_loop) { }
-    test_ttt.game_loop
+    expect(test_ttt.game_loop).to receive(:game_loop).once
+    expect(game_io).to receive(:puts_message)
+    expect(game_io).to receive(:get_input).and_return("N")
+    test_ttt.play_game
+  end
+
+  it "plays the game twice " do
+    expect(test_ttt.game_loop).to receive(:game_loop).twice
+    allow(test_ttt).to receive(:new_game)
+    expect(game_io).to receive(:puts_message).twice
+    expect(game_io).to receive(:get_input).and_return("Y", "N")
+    test_ttt.play_game
+  end
+
+  it "gameboad is blank after new_game " do
+    expect(test_ttt.game_loop).to receive(:game_loop).twice
+    expect(test_ttt).to receive(:play_again?).and_return(true, false)
```

Comment:
Again, stubbing variables/methods on the thing under test is usually a smell. If you're injecting the `game_io` and that's the thing that ultimately controls whether or not a game is played again, then you would probably just stub that.